RecoLocalTracker/Phase2TrackerRecHits/python/Phase2TrackerRecHits_cfi.py

```diff
@@ -3,7 +3,5 @@
 # RecHits options
 siPhase2RecHits = cms.EDProducer("Phase2TrackerRecHits",
   src = cms.InputTag("siPhase2Clusters"),
-  Phase2StripCPE = cms.ESInputTag("phase2StripCPEESProducer", "Phase2StripCPEDummy")
+  Phase2StripCPE = cms.ESInputTag("phase2StripCPEESProducer", "Phase2StripCPETrivial")
```

Comment:
Where is the Phase2StripCPEGeometric going to be used?
I thought that the hit reco is supposed to change to it already